packages/react-native-payments/src/class/payment-response/ios-payment-response.ts

```diff
@@ -52,7 +52,7 @@ export class IosPaymentResponse extends PaymentResponse {
     }
 
     private static parseNSPersonNameComponents(input?: IosNSPersonNameComponents): string {
-        return [input?.familyName, input?.middleName, input?.givenName].filter(isNotEmptyString).join('');
+        return [input?.familyName, input?.middleName, input?.givenName].filter(isNotEmptyString).join(',');
```

Comment:
```ts
return [input?.familyName, input?.middleName, input?.givenName].filter(isNotEmptyString).join(' ');
```

Reply:
The issue with using space as a separator is that some people have two parts of their first or last name. For example Vincent van Gogh. With space there is no way for me to know if van is part of the first name, the middle name, or part of the last name.